schema.graphql

```diff
@@ -1,67 +1,25 @@
-type TX @entity {
-  id: ID!
-  createdAt: String!
-}
-# TOP HATS MAY BE UNNECESSARY
-# Entity that recursively maps an TopHat's entire tree
-type TopHatTree @entity {
-  id: ID!
-  createdAt: String!
-  owner: Bytes!
-  subHats: [HatTree!]
-}
-# Finite TopHat data
-type TopHat @entity {
-  id: ID!
-  createdAt: String!
-  owner: Bytes!
-  subHats: [HatTree!]
-}
-# Entity that recursively maps an Hat's tree
-type HatTree @entity {
```

Comment:
Curious what you think about the utility of this entity. What would it enable for front ends? Do you think it makes sense to bring something like it back in a subsequent version?